### ISSUE TITLE
How to upload image to server?

### ISSUE THREAD

**Issue by NitinChronext** [mrousavy/react-native-jsi-image#4]
I Liked the lib but there is no clear documentation about how we can upload images (UIImage ) to the server. It only shows how we can download it

Thanks,
Nitin

**Comment by mrousavy** [comment]
Hey - this could be implemented in a third party API since there's so much stuff to an image upload function (API params, authentication, HTTP headers, buffering, etc etc)

**Comment by NitinChronext** [comment]
No I am saying How can we send UIImage object to https since your point was not to convert image into base64

**Comment by mrousavy** [comment]
Yes, we are talking about the same thing. In this case you will have to upload a binary stream, or base64. But there currently is no API for this, this needs to be implemented.

**Comment by lukebrandonfarrell** [comment]
I think it would be out of scope of this library to handle upload. We need to give Turbo modules and Fabric some time to mature. Ideally, I'd see a flavour of Expo uploadAsync made to handle JSI image, but Expo still haven't implemented Fabric or Turbo modules.
